sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeHealthcareEntitiesOptions.java

```diff
@@ -43,8 +43,7 @@ public AnalyzeHealthcareEntitiesOptions setIncludeStatistics(boolean includeStat
     /**
      * Set the value of {@code disableServiceLogs}.
      *
-     * @param disableServiceLogs The default value of this property is 'false', except for methods like
-     * 'beginAnalyzeHealthcareEntities' and 'recognizePiiEntities'. This means, Text Analytics service logs
+     * @param disableServiceLogs The default value of this property is 'true'. This means, Text Analytics service logs
```

Comment:
hmm well now that it is true the message could be something like:
The default value of this property is 'true'. This means Text Analytics service doesn't log your input text. Setting this property to 'false', enables the service to log your text input for 48 hours, solely to allow for troubleshooting issues

Reply:
updated.